conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/CompressionFilter.java

```diff
@@ -1,165 +0,0 @@
-/*
- * (c) Copyright 2017 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.conjure.java.server.jersey;
-
-import com.google.common.cache.CacheBuilder;
-import com.google.common.cache.CacheLoader;
-import com.google.common.cache.LoadingCache;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.net.HttpHeaders;
-import com.google.common.net.MediaType;
-import com.jcraft.jzlib.DeflaterOutputStream;
-import com.jcraft.jzlib.GZIPOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.util.List;
-import java.util.function.Function;
-import javax.ws.rs.container.ContainerRequestContext;
-import javax.ws.rs.container.ContainerResponseContext;
-import javax.ws.rs.container.ContainerResponseFilter;
-import javax.ws.rs.core.MultivaluedMap;
-
-/**
- * Compresses responses based on the Accept-Encoding header in the request.
- * <p>
- * This is an alternative compression filter implementation that uses jzlib for compression rather than the java
- * standard library due to performance issues observed on high core machines:
- * https://bugs.openjdk.java.net/browse/JDK-8179001
- *
- * @deprecated This class is unused and will be considered for removal in the next major release. As a workaround,
- * consider setting the Jetty compression threshold to a small value (e.g., 1MB).
```

Comment:
@uschi2000, can you clarify  this comment?

Reply:
we're not using this anywhere, check the internal wc codepath.